CHANGELOG.md

```diff
@@ -59,6 +59,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   two fixed low and high dimensional prior regimes
 - The previous default kernel factory has been renamed to `EDBOKernelFactory` and now
   fully reflects the original logic
+- Regular "EI" acquisition functions are mapped to their "log" versions for improved
+  numerical stability
```

Comment:
somewhere in the docs this implicit change has to be mentioned, perhaps as admonition on recommenders page

Reply:
Fully agree, IF we decide to include this (see my other comment) this should be mentioned somewhere as it might be unexpected.

Reply:
Since we only changed the defaults now, [no mention required](https://github.com/emdgroup/baybe/pull/326#discussion_r1698118943) (apart from changelog). Once we have a proper ACQF user guide, we'll add the info there 👍🏼